packages/@lwc/compiler/src/transformers/__tests__/transform-html.spec.ts

```diff
@@ -60,46 +60,29 @@ describe('transformSync', () => {
         expect(code).toContain(`tmpl.stylesheetToken = "lwc-143n22jptum";`);
     });
 
-    it('should hoist static vnodes when enableStaticContentOptimization is set to true', () => {
-        const template = `
-            <template>
-                <img src="http://www.example.com/image.png" crossorigin="anonymous">
-            </template>
-        `;
-        const { code, warnings } = transformSync(template, 'foo.html', {
-            enableStaticContentOptimization: true,
-            ...TRANSFORMATION_OPTIONS,
-        });
-
-        expect(warnings).toHaveLength(0);
-        expect(code).toMatch('<img');
-    });
-
-    it('should not hoist static vnodes when enableStaticContentOptimization is set to false', () => {
-        const template = `
-            <template>
-                <img src="http://www.example.com/image.png" crossorigin="anonymous">
-            </template>
-        `;
-        const { code, warnings } = transformSync(template, 'foo.html', {
-            enableStaticContentOptimization: false,
-            ...TRANSFORMATION_OPTIONS,
+    describe('enableStaticContentOptimization', () => {
```

Comment:
nit: so that the error message shows up nicely.
```suggestion
    describe('enableStaticContentOptimization: ', () => {
```